Source/ConversationV1/Models/DialogNode.swift

```diff
@@ -175,18 +179,18 @@ extension DialogNode: Codable {
     public func encode(to encoder: Encoder) throws {
         var container = encoder.container(keyedBy: CodingKeys.self)
         try container.encode(dialogNodeID, forKey: .dialogNodeID)
-        try container.encodeIfPresent(description, forKey: .description)
-        try container.encodeIfPresent(conditions, forKey: .conditions)
-        try container.encodeIfPresent(parent, forKey: .parent)
-        try container.encodeIfPresent(previousSibling, forKey: .previousSibling)
-        try container.encodeIfPresent(output, forKey: .output)
-        try container.encodeIfPresent(context, forKey: .context)
-        try container.encodeIfPresent(metadata, forKey: .metadata)
-        try container.encodeIfPresent(nextStep, forKey: .nextStep)
+        try container.encode(description, forKey: .description)
+        try container.encode(conditions, forKey: .conditions)
+        try container.encode(parent, forKey: .parent)
+        try container.encode(previousSibling, forKey: .previousSibling)
+        try container.encode(output, forKey: .output)
+        try container.encode(context, forKey: .context)
+        try container.encode(metadata, forKey: .metadata)
+        try container.encode(nextStep, forKey: .nextStep)
         try container.encode(created, forKey: .created)
-        try container.encode(updated, forKey: .updated)
+        try container.encodeIfPresent(updated, forKey: .updated)
         try container.encodeIfPresent(actions, forKey: .actions)
-        try container.encodeIfPresent(title, forKey: .title)
+        try container.encode(title, forKey: .title)
```

Comment:
Making the properties optional in `staging/conversation` generated this code with `decodeIfPresent` and `encodeIfPresent`. That accounts for all of these changes.